introduction.md

```diff
@@ -1,3 +1,7 @@
 My name is Marcin Sieka. I work for Hewlett-Packard as a Programmer. I am 32 years old. 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?